DigraphyApi/appsettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DigraphyDataBase": "Host=ep-calm-shape-a7f78a2g.ap-southeast-2.aws.neon.tech;Database=DigraphyDB;Username=neondb_owner;Password=V4P8OIeoFbcA;SSL Mode=Require;Trust Server Certificate=true"
```

Comment:
UHHH This should not be committed :sob:

You're going to want to change the credentials for your database.

You can set the credential using `dotnet user-secrets set "ConnectionStrings:DigraphyDataBase" "Your database connection string"`.

Also can it be `DigraphyDatabase` with a lowercase `b` :)